extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/__init__.py

```diff
@@ -50,11 +53,12 @@
 
 REMOTE_INFERENCE_CLASSES = [
     "HuggingFaceImage2TextRemoteInference",
+    "HuggingFaceConversationalRemoteInference",
     "HuggingFaceText2ImageRemoteInference",
     "HuggingFaceText2SpeechRemoteInference",
     "HuggingFaceTextGenerationRemoteInference",
     "HuggingFaceTextSummarizationRemoteInference",
     "HuggingFaceTextTranslationRemoteInference",
-    "HuggingFaceAutomaticSpeechRecognitionRemoteInference"
+    "HuggingFaceAutomaticSpeechRecognitionRemoteInference",
 ]
 __ALL__ = LOCAL_INFERENCE_CLASSES + REMOTE_INFERENCE_CLASSES + UTILS
```

Comment:
Just checking -- is local inference with conversational pipeline significantly different?

Reply:
local and inference have different api completion parameters. It won't be entirely the same; unsure if its a good use of time